### PR TITLE
wmco/wmcb: Add CI jobs for community-4.9

### PR DESCRIPTION
**Pull request by alinaryan**
- wmco/wmcb: Add configs for community-4.9
- wmco/wmcb: conditionally skip CI jobs for community-4.9
- core-services: onboard community-4.9 branch

### CODE REVIEW

**Comment by alinaryan** [comment]
\hold

File: ci-operator/jobs/openshift/windows-machine-config-bootstrapper/openshift-windows-machine-config-bootstrapper-community-4.9-presubmits.yaml [hunk 1]
```diff
@@ -16,6 +16,7 @@ presubmits:
       pj-rehearse.openshift.io/can-be-rehearsed: "true"
     name: pull-ci-openshift-windows-machine-config-bootstrapper-community-4.9-okd-build
     rerun_command: /test okd-build
+    skip_if_only_changed: ^docs/|\.md$|^(?:.*/)?(?:\.gitignore|OWNERS|PROJECT|LICENSE)$
```
Comment (sebsoto) [inline ci-operator/jobs/openshift/windows-machine-config-bootstrapper/openshift-windows-machine-config-bootstrapper-community-4.9-presubmits.yaml:19]: This needs to be done for the community-4.9 WMCO branch as well

**Review by sebsoto — CHANGES_REQUESTED** [review]

File: ci-operator/jobs/openshift/windows-machine-config-bootstrapper/openshift-windows-machine-config-bootstrapper-community-4.9-presubmits.yaml (hunk 1 above)
Comment (alinaryan) [reply ci-operator/jobs/openshift/windows-machine-config-bootstrapper/openshift-windows-machine-config-bootstrapper-community-4.9-presubmits.yaml:19]: Added

File: ci-operator/jobs/openshift/windows-machine-config-operator/openshift-windows-machine-config-operator-community-4.9-presubmits.yaml [hunk 2]
```diff
@@ -0,0 +1,265 @@
+presubmits:
+  openshift/windows-machine-config-operator:
+  - agent: kubernetes
+    always_run: true
```
Comment (jrvaldes) [inline ci-operator/jobs/openshift/windows-machine-config-operator/openshift-windows-machine-config-operator-community-4.9-presubmits.yaml]: Consider introducing https://github.com/openshift/release/pull/23899/commits/dade17520eeb7f2eb96341d5b050b4786332ae69 as part of this commit..

**Review by jrvaldes — COMMENTED** [review]
@alinaryan  thanks for working on this, comments added. PTAL

File: ci-operator/jobs/openshift/windows-machine-config-bootstrapper/openshift-windows-machine-config-bootstrapper-community-4.9-presubmits.yaml [hunk 3]
```diff
@@ -1,7 +1,7 @@
 presubmits:
```
Comment (saifshaikh48) [inline ci-operator/jobs/openshift/windows-machine-config-bootstrapper/openshift-windows-machine-config-bootstrapper-community-4.9-presubmits.yaml:1]: commit message missing values starting + ending with values: `doc/` and `.md`

Comment (saifshaikh48) [inline ci-operator/jobs/openshift/windows-machine-config-bootstrapper/openshift-windows-machine-config-bootstrapper-community-4.9-presubmits.yaml:1]: I suggest squashing this commit into cecf1e13ea940f7a75fa29613a5f2ef00637fe36 to reduce code churn

File: ci-operator/jobs/openshift/windows-machine-config-operator/openshift-windows-machine-config-operator-community-4.9-presubmits.yaml [hunk 4]
```diff
@@ -1,7 +1,7 @@
 presubmits:
```
Comment (saifshaikh48) [inline ci-operator/jobs/openshift/windows-machine-config-operator/openshift-windows-machine-config-operator-community-4.9-presubmits.yaml:1]: Same comments, edit commit message and squash into 5da16afcabbc3525eba8294f664029dd735990d9

**Review by saifshaikh48 — COMMENTED** [review]
@alinaryan Thanks for working on this, left a few suggestions

File: ci-operator/jobs/openshift/windows-machine-config-bootstrapper/openshift-windows-machine-config-bootstrapper-community-4.9-presubmits.yaml (hunk 3 above)
Comment (alinaryan) [reply ci-operator/jobs/openshift/windows-machine-config-bootstrapper/openshift-windows-machine-config-bootstrapper-community-4.9-presubmits.yaml:1]: Thanks completely missed that - fixed.

**Comment by alinaryan** [comment]
/retest ci/rehearse/openshift/windows-machine-config-bootstrapper/community-4.9/okd-e2e-wmcb
Hybrid Overlay timeout

**Comment by alinaryan** [comment]
/test pj-rehearse
Retest required because of timeout

**Comment by alinaryan** [comment]
/test app-ci-config-dry
retest required as step-registry has updated.

**Pull request merged**